PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)
+
+[//]: # (Put Issue # here, if applicable)
+Fixes #______
```

Comment:
Can this be an instruction to copy the issue's URL? That removes the chance of people mistyping the issue number.